skymp5-server/cpp/papyrus_vm_lib/Reader.cpp

```diff
@@ -13,7 +14,9 @@ void Reader::Read()
   if (File.is_open()) {
 
     File.seekg(0, std::ios_base::end);
+    fl_size = File.tellg();
```

Comment:
I think there's no reason to define it in the beginning, could say `const auto fileSize = ...` right here.